wisdem/floatingse/member.py

```diff
@@ -72,9 +71,6 @@ class DiscretizationYAML(om.ExplicitComponent):
     E_mat : numpy array[n_mat, 3], [Pa]
         2D array of the Youngs moduli of the materials. Each row represents a material,
         the three members represent E11, E22 and E33.
-    E_user : float, [Pa]
```

Comment:
Is this a purposeful removal? I think this is in current develop (and was used for the linearization studies). Maybe this PR just needs develop pulled in as well.

Reply:
Not a purposeful renewal.  Likely more a merge conflict resolution mistake.  There were quite a bit of those between WISDEM & WEIS recently.  Do you want to restore the line or should I?

Anything else fall in that category?

Reply:
Got it, sorry for the mix-up on the merging. Please restore the line; that's the only issue I see on this PR.